hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemAppend.java

```diff
@@ -427,9 +427,9 @@ public void testParallelWriteDifferentContentLength() throws Exception {
    **/
   @Test
   public void testParallelWriteOutputStreamClose() throws Exception {
-    Assume.assumeTrue(getFileSystem().getAbfsStore().getPrefixMode() == PrefixMode.BLOB);
-    final Path SECONDARY_FILE_PATH = new Path("secondarytestfile");
     AzureBlobFileSystem fs = getFileSystem();
+    Assume.assumeTrue(!OperativeEndpoint.OperativeEndpointFallback.isIngressEnabledOnDFS(getPrefixMode(fs), getAbfsStore(fs).getAbfsConfiguration()));
```

Comment:
it doesn't translate to `Assume.assumeTrue(getFileSystem().getAbfsStore().getPrefixMode() == PrefixMode.BLOB);`

Reply:
what if its blob bt ingressFallbackToDfs is false?

Reply:
I think i got my above comment wrong. What i meant was for:
```
if (mode == PrefixMode.BLOB) {
                return abfsConfiguration.shouldIngressFallbackToDfs();
            } else return mode == PrefixMode.DFS;
```
if shouldIngressFallbackToDfs is true, although its in Blob, the assumption will fail.

Reply:
Yes if ingressFallback is true in blob, we don't want this test to run. This is mainly for PutBlock and PutBlockList verification.

Reply:
okay. So the condition in ABFS_3.3.2_dev is wrong?

Reply:
okay. But in ABFS_3.3.2_dev, it checks only Blob endpoint. So, the assumption in the pr would get different?

Reply:
Yes in that branch if you make the ingress config to true, this test case will fail